frontend/src/types.ts

```diff
@@ -319,6 +319,17 @@ export type PodContainer = {
   volumeMounts?: VolumeMount[];
 };
 
+export type PodStepState = {
+  stepName: string;
+  state: StepState;
+};
+
+export enum StepState {
```

Comment:
nit, make state more specific as it is in our root types file so maybe `PodStepStateType`? or a long those lines

Reply:
done